bin/pycbc_compress_bank

```diff
@@ -157,6 +156,8 @@ for ii in range(imin, imax):
     htilde = bank[ii-imin]
     tmplt = bank.table[ii-imin]
     fmin=tmplt.f_lower
+    template_duration=htilde.chirp_length
```

Comment:
Oh, nevermind. I just dug deeper and realized that `get_waveform_filter` does set the `chirp_length` argument. So you can ignore this comment.